tracer/build/_build/Build.Steps.cs

```diff
@@ -639,23 +639,50 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 .EnableNoRestore()
                 .CombineWith(targetFrameworks, (p, framework) => p
                     .SetFramework(framework)
-                    .SetOutput(MonitoringHomeDirectory / framework)));
+                    .SetOutput(MonitoringHomeDirectory / framework))
+            );
         });
 
+    Target PublishManagedTracerR2R => _ => _
+        .Unlisted()
+        .After(CompileManagedSrc)
+        .Executes(() =>
+        {
+            var targetFramework = TargetFramework.NET6_0;
+
+            // Needed as we need to restore with the RuntimeIdentifier
+            DotNetRestore(s => s
+                .SetProjectFile(Solution.GetProject(Projects.DatadogTraceMsBuild))
```

Comment:
Do you really need this at _all_? 🤔 I'm about 90% sure you don't need this dll at all, and _should_ be stripping it out of the layer. @tonyredondo can you confirm?

Reply:
It's stripped down later, but I want to make sure I build as close as possible to the current process, don't want to diverge much in case in the future we need to use some r2r binaries for other platforms like Azure!

Reply:
Just for info, I believe this is _literally_ only used if you're using the tracer as _part_ of your build process, so I think there's basically no chance you'll need it 😃